torch/csrc/jit/passes/onnx/constant_fold.cpp

```diff
@@ -0,0 +1,187 @@
+#include <torch/csrc/jit/passes/onnx/constant_fold.h>
+#include <c10/util/Exception.h>
+
+#include <c10/util/Optional.h>
+#include <algorithm> 
+
+#if defined(_MSC_VER)
+#include <BaseTsd.h>
+typedef SSIZE_T ssize_t;
+#endif
+
+namespace torch {
+namespace jit {
+
+namespace onnx {
+using namespace ::c10::onnx;
```

Comment:
This is so that the ops are identified with small_ish_ but still qualified names such as onnx::Slice etc. Mostly to improve code readability. Also other passes follow similar style.